clients/fides-js/src/lib/consent.tsx

```diff
@@ -39,10 +41,12 @@ export const initOverlay = async ({
         document.body.prepend(parentElem);
       }
 
+      // DEFER: use a separate TCF bundle instead of this if statement
```

Comment:
okay SO the main idea here is that there is a different entrypoint component for TCF vs Notices. Rather than just our handy `Overlay` component, we now have `NoticeOverlay` and `TcfOverlay`, both of which wrap the original `Overlay` component. this is handy for code splitting, because (once we move away from the if/else logic, and instead have `fides-tcf.ts` render `TcfOverlay`) rollup should be able to figure out which components we need for each bundle